internal/pkg/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Repository string `yaml:"repository,omitempty"`
+	Username   string `yaml:"username,omitempty"`
+}
+
+func Open() (*Config, error) {
+	c := &Config{}
```

Comment:
not sure I care about it for initial PR, but we should add support for specifying the bldr.yaml location